Dockerfile-dev

```diff
@@ -18,13 +18,17 @@ RUN <<EOF
   set -exo pipefail
   apt-get update
   apt-get -y upgrade
-  apt-get install -y --no-install-recommends libmagic1 gettext graphviz graphviz-dev chromium
+  apt-get install -y --no-install-recommends libmagic1 gettext graphviz graphviz-dev chromium npm
+  npm install -g @lhci/cli@0.14.x
   pip install --upgrade pip
   pip install -r requirements-dev.txt
 EOF
 
 # Copy project
 COPY ./app /app/
 
+# Used in LHCI
+COPY entrypoint.sh /
```

Comment:
Please put this `COPY` above the `COPY app` line, as that one is more likely to result in changes. That way we are more likely to obtain reuse of the docker layer cache.  (Also wrong in `Dockerfile`.)

Reply:
can also just go in a separate PR if you prefer.